File: 002-Backend/LaPlata.API/LaPlata.Domain/DTOs/Despesas/Conta/ReadContaDTO.cs
namespace LaPlata.Domain.DTOs
{
    public class ReadContaDTO
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public int ValorInteiro { get; set; }
        public int? ValorCentavos { get; set; }
        public int DiaVencimento { get; set; }
        public ReadCategoriaDTO Categoria { get; set; }
        public List<ReadPagamentoContaDTO> Pagamentos { get; set; }
    }
}

File: 003-Frontend/HomeTools.Front/src/app/interfaces/api-dto/Financas/readContaDto.ts
import { Categoria } from "../../categoria";
import { PagamentoContaDto } from "./readPagamentoContaDto";

export interface ReadContaDto {
  Id: number;
  Descricao: string;
  ValorInteiro: number;
  ValorCentavos: number;
  DiaVencimento: number;
  Categoria: Categoria;
  Pagamentos: PagamentoContaDto[];
}
